src/lib/errorHandler.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  Request,
+  Response,
+  NextFunction,
+} from "../../node_modules/@types/express"
```

Comment:
This can be just “@types/express”

Reply:
Ah, cool, something seemed off here!

Reply:
Weird, this actually failed without the relative path:
```
error TS6137: Cannot import type declaration files. Consider importing 'express' instead of '@types/express'.
```

Reply:
Doh. Yeah you should import just from `express`, my bad. TS merges `@types/*` into the actual packages and its modules.